src/state.js

```diff
@@ -0,0 +1,21 @@
+import { startWorker } from "./display/highlight_worker";
```

Comment:
This one's name doesn't say much about its content. Maybe move it into a bigger utility module, or call it mode_state.js